app/root.css

```diff
@@ -28,6 +28,15 @@
 
   --colors-green-500: #40c075;
 
+  /* Dark mode semantic colors */
+  --background: #ffffff;
```

Comment:
wut? I guess comment can be deleted, it probably tried to explain something but it ended up confusing/useless